### ISSUE TITLE
Clarification on Issue Update

### ISSUE THREAD

**Issue by leesoh** [ncatlabs/pydradis#1]
Hi there,

First off, thanks for building this! I'm excited to put it to use. I have a quick question regarding the update_issue method. Does it currently only support updating title and description? If it supports updating all fields in an issue, can you provide an example of what that would look like?

**Comment by leesoh** [comment]
Doing something like this seems to work, but it's hacky:

```
pd.update_issue(project_id, issue_id, 'This is a title', 'Description is here\r\n\r\n#[Header]#\r\nhiiii')
```
results in:

```
#[Title]#
This is a title

#[Description]#
Description is here

#[Header]#
hiiii
```

**Comment by pmsosa** [comment]
Thats initially how it's meant to be used as there are no assumptions as to what fields you are specifying, thus giving you the flexibility to define whatever body you want.

Perhaps we could look into changing the update method such that one doesn't have to take care of the  formatting but can simply define a dictionary of fields & their accompanying text.

**Comment by leesoh** [comment]
Yeah, that's what I was thinking as well. Looking at the Dradis API documentation, the issue update method   expects a string ('#[Title]#...'), although the get issue's `field` key contains a dict that would be lovely to just send back modified, but as it is today it looks like it would need to be parsed into a string first.

**Comment by leesoh** [comment]
Took a stab at it here: https://github.com/leesoh/pydradis/blob/dev/pydradis.py#L560
